mappings/net/minecraft/entity/EntityWithAiGoals.mapping

```diff
@@ -1,4 +1,17 @@
 CLASS us net/minecraft/entity/EntityWithAiGoals
-	FIELD a pos Ldx;
+	FIELD a aiCenter Ldx;
```

Comment:
Generally I believe this is referred to as a "home", assuming this is what I think it is.

<*insert tirade about Scuttlebugs*>

`aiCenter` → `home`?